Containers/borgbackup/borg_excludes

```diff
@@ -0,0 +1,9 @@
+nextcloud_aio_volumes/nextcloud_aio_apache/caddy/
+nextcloud_aio_volumes/nextcloud_aio_mastercontainer/caddy/
+nextcloud_aio_volumes/nextcloud_aio_nextcloud/data/nextcloud.log*
+nextcloud_aio_volumes/nextcloud_aio_nextcloud/data/audit.log
+nextcloud_aio_volumes/nextcloud_aio_mastercontainer/certs/
+nextcloud_aio_volumes/nextcloud_aio_mastercontainer/data/daily_backup_running
+nextcloud_aio_volumes/nextcloud_aio_mastercontainer/data/session_date_file
+nextcloud_aio_volumes/nextcloud_aio_mastercontainer/session/
+nextcloud_aio_volumes/nextcloud_aio_mastercontainer/data/id_borg*
```

Comment:
we have this now in 3 places IIRC. can we maybe somehow merge this so that we only need to maintain one place?

Reply:
All 3 places have a different syntax, there's no common format, e.g. rsync only supports its own format. I would have to generate the patterns, which seems tricky in bash; it would take me a while to figure out and I think might make the code harder to read, especially for the `find`. For example:

rsync

```
        --exclude "nextcloud_aio_mastercontainer/caddy/**" \
        --exclude "nextcloud_aio_nextcloud/data/audit.log" \
```

vs find

```
                        -o -path nextcloud_aio_volumes/nextcloud_aio_mastercontainer/caddy \
                        -o -path "nextcloud_aio_volumes/nextcloud_aio_mastercontainer/caddy/*" \
                        -o -path nextcloud_aio_volumes/nextcloud_aio_nextcloud/data/audit.log \
```

vs borg

```
nextcloud_aio_volumes/nextcloud_aio_mastercontainer/caddy/
nextcloud_aio_volumes/nextcloud_aio_nextcloud/data/audit.log
```

Reply:
I see. Then please add a comment to all 3 places that we need to keep them in sync

Reply:
Added